src/utils/ledger.ts

```diff
@@ -29,13 +28,12 @@ export const ledgerErrorHandler = (error: Error) => {
   ) {
     return LEDGER_ERROR_CODES.OFF_OR_LOCKED;
   }
+  if (error.name.includes('Disconnected')) {
```

Comment:
curious if there's a better way to identify ledger error types, i'm guessing you've looked into it haha. they don't provide error types out of the box?